sites/hashdev/src/_pages/blog/1_block-design.mdx

```diff
@@ -0,0 +1,255 @@
+
+It comes with a default palette of colors you can use, a typographic scale, a spacing scale, and other similar base styles. You can read more details about what the base stylesheet includes in our [documentation](https://blockprotocol.org/spec/block-types#styling).
+
+### Accepting styles from embedding applications
+
+The blocks you build will be usable inside other applications; we call these "embedding applications."
+
+The protocol gives embedding applications control over the way blocks appear within them. We give embedding applications the final say over how a block looks, as they have more context about the environment it lives within and how people will use it.
+
+Every Block Protocol block you build should accept a `styleVariables` object that the embedding application can pass in.
+
+This object defines a set of variables that control the basic styles of the block. We've designed these to maximize consistency and simplicity within your block. You can see the full list of these in the [styling section](https://blockprotocol.org/spec/block-types#styling) of our documentation. Here's a rundown of the major categories:
+
+1.  Colors - a range of shades for neutrals, primary, secondary, tertiary, warning, and danger.
+1.  Typography – font family, base size, scaling ratios, and line heights
+1.  Spacing – base size and scaling ratios
+1.  Borders – weight, colors, and radius
+1.  Box shadows
+1.  Icon sizes
+
+These styling variables can do a lot of the heavy lifting to help blocks match their surrounding environment. As a block developer, you should make sure the markup in your blocks uses these styling variables to make life easier for the people using your blocks.
+
+Embedding applications can also pass a plain old CSS file into the block if they need to manually override specific styles and layouts. This should be a last resort though, as futzing with CSS selectors can get messy and may end up affecting the core functionality of the block.
+
+## Usability and UX principles
+
+No matter how beautiful a block looks, it's not worth much to anyone if it's difficult to use. Soft drop shadows and smooth gradients aren't going to delight the user if they're furiously trying to find the “save” button, and you've stashed it inside a dropdown menu that's too low contrast to see.
+
+<img
+  src="blog/1_block-design/DesignGuidelines_1.png"
+  width={900}
+  height={480}
+  alt="A text input field with a very low contrast dropdown menu below it. Inside there is a low contrast 'save' button and a much larger and more prominent 'delete' button."
+/>
+
+Usability and user experience design (UX) is a whole discipline in its own right, and we can't cover everything in this guide. We're not expecting you to become a UX designer before you can start building blocks. However, knowing the basics of UX and some of the most important principles will go a long way in making you a better block builder.
+
+#### 1. Keep it vanilla
```

Comment:
@MaggieAppleton we have a linting rule that prevents skipping header levels - can these be level 3 headers? is it that we want the level 3 headers to be a bit smaller?

I can turn off the linting rule, but it does make more semantic sense to have these as level 3 – if it's a styling issue we can fix that instead